src/components/nodes/node-stanford-person.tsx

```diff
@@ -20,104 +21,158 @@ export const NodeStanfordPerson = ({node, ...props}: PersonNodeProps) => {
+        <div className="sm:su-flex su-no-wrap su-rs-mb-5 su-mt-50">
+          <Conditional showWhen={node.su_person_photo.field_media_image}>
+            <div className="su-rs-mr-4">
+                <div className="su-rounded-full su-w-[220px] su-h-[220px] su-overflow-hidden">
+                    <DrupalImage
+                        src={node.su_person_photo.field_media_image.image_style_uri.medium_square}
+                        alt={node.su_person_photo.field_media_image.resourceIdObjMeta.alt}
+                        height={node.su_person_photo.field_media_image.resourceIdObjMeta.height}
+                        width={node.su_person_photo.field_media_image.resourceIdObjMeta.width}
+                    />
+                </div>
+            </div>
+          </Conditional>
 
           <div>
-            {node.su_person_short_title}
+            <Conditional showWhen={node.su_person_short_title}>
+              <div className="su-type-0 su-leading">{node.su_person_short_title}</div>
+            </Conditional>
             <h1>{node.title}</h1>
-            {node.su_person_full_title}
+            <Conditional showWhen={node.su_person_full_title}>
+              <div className="su-type-0 su-leading">{node.su_person_full_title}</div>
+            </Conditional>
+            <Conditional showWhen={node.su_person_pronouns}>
+              <div className="su-type-0 su-leading">Pronouns: {node.su_person_pronouns}</div>
+            </Conditional>
           </div>
         </div>
 
         <div className="md:su-grid su-grid-cols-6 su-gap-[40px]">
           <div className="su-col-span-4">
-            {node.body?.processed && <div>{formatHtml(node.body.processed)}</div>}
-
-            {node.su_person_components && node.su_person_components.map(paragraph =>
-              <Paragraph key={paragraph.id} paragraph={paragraph}/>
-            )}
-
-            {node.su_person_education &&
-                <div>
-                    <h2>Education</h2>
-                  {node.su_person_education}
-                </div>
-            }
+            {node.body?.processed && <div className="su-type-1 su-rs-mt-6 sm:su-rs-mt-0 su-rs-mb-7 md:su-w-10/12 ">{formatHtml(node.body.processed)}</div>}
+            
+            <Conditional showWhen={node.su_person_components}>
+              {node.su_person_components.map(paragraph =>
+                <Paragraph key={paragraph.id} paragraph={paragraph}/>
+              )}
+            </Conditional>
+
+            <Conditional showWhen={node.su_person_education}>
+              <div className="su-rs-mb-7">
+                <h2 className="su-type-0">Education</h2>
+                {node.su_person_education.map((education, index) =>
+                  <div key={`person-education-${index}`} className="su-rs-mb-0">
+                    { education }
+                  </div>
+                )}
+              </div>
+            </Conditional>
+
+            <Conditional showWhen={node.su_person_research}>
```

Comment:
It looks like the fields that allow multiple (the ones you have to do `.map` to) need to have the conditional something like `node.su_person_research.length > 0`

See the sample content:
https://deploy-preview-41--stanford-library.netlify.app/people/dena-debry